### DISCUSSION TITLE
Can I execute another package commands via agent?

### DISCUSSION THREAD

**Discussion by mraguth** [microsoft/autogen#873] [Q&A] [answered]
I am currently running tests with Autogen in a practical scenario. I have successfully established a connection with the AzureDatabase, and managed to extract data from several tables. Is it feasible for me to use a package named 'panadasai' on my retrieved dataframe? Additionally, would it be plausible to construct an agent and instruct it to implement 'pandasai' commands such as 'chat'? Thank you for your help.   

Pandasai pacakge: https://github.com/gventuri/pandas-ai    (i like to call this package method 'chat' via agen)

**Comment by gagb** [comment, marked as answer]
Yes and this notebook maybe useful. Try adapting it to your `pandasai` use case: https://github.com/microsoft/autogen/blob/main/notebook/agentchat_auto_feedback_from_code_execution.ipynb
